website/content/en/highlights/2022-05-03-0-22-0-upgrade-guide.md

```diff
@@ -63,3 +64,40 @@ project_id = "vector-123456"
 ```
 
 For more information on the new syntax, you can review the documentation [here](https://vector.dev/docs/reference/configuration/sinks/gcp_stackdriver_metrics/)
+
+
+#### [VRL now supports template strings] {#vrl-template-strings}
+
+VRL strings can now be templated. It is now possible to insert the values of
+variables by inserting a placeholder with an embedded variable name into the
+string using `{{..}}`. For example in the following code:
+
+```coffee
+beverage = "coffee"
+preference = "I love to drink {{ beverage }}!"
+
+assert!(preference == "I love to drink coffee!")
+```
+
+It should be noted that currently the placeholder must contain a simple
+variable name and that variable must resolve to a string.
+
+This will not work:
+
+```coffee
+stars = 42
+sky = "There are {{ stars }} in the sky."
+```
```

Comment:
It's always good to offer solutions as part of the example after saying something will not work, my suggestion:


> This will not work:
> 
> 
> ```coffee
> stars = 42
> sky = "There are {{ stars }} in the sky."
> ```
> 
> 
> Instead, do this:
>
> ```coffee
> stars = to_string!(42)
> sky = "There are {{ stars }} in the sky."
> ```